### ISSUE TITLE
Easier to read/understand error code(s)

### ISSUE THREAD

**Issue by Bramvdhout** [CLARIAH/COW#40]
While converting a .csv with a .nq already present, the following error is shown:
![image](https://user-images.githubusercontent.com/32885158/35215782-ac48ed06-ff65-11e7-82a2-e060c27faa95.png)

Already established in Issue #37 that this is caused by the .nq being present. However, this error message is not clear (enough): maybe reword it (and possibly others) for user convenience?

**Comment by melvinroest** [comment]
For me, the error was clear enough in Power Shell. This is what I got
![image](https://user-images.githubusercontent.com/44729293/95752220-fd10c480-0c9f-11eb-8f22-63f8c3b450fd.png)

It's quite clear that `\` is changed to `_`.

Closing this issue.
